modules/payment/src/Entity/PaymentMethod.php

```diff
@@ -224,6 +224,29 @@ public function setCreatedTime($timestamp) {
   /**
    * {@inheritdoc}
    */
+  public function getOwnerRemoteId() {
```

Comment:
i also thought the same initially. but working on it looks like PaymentMethod entity methods